src/main/java/com/toptrumps/cli/CommandLineUI.java

```diff
@@ -78,8 +79,9 @@ public void requestNumberOfPlayers() {
         }
     }
 
-    public void showPlayerCard() {
-
+    public void showPlayerCard(Card c) {
+        System.out.println("You drew \'" + c.getDescription() + "\':");
+        System.out.println(c.stringAttributes());
```

Comment:
I would combine these print statements within the Card class instead of calling 2 separate statements here. Possibly as a toString?

Reply:
So, that's what I was going to do initially. But if you look at the example CLI out put it loops the attributes a number of times seperately. I was trying to follow DRY.
  
![Capture](https://user-images.githubusercontent.com/20328038/73566778-9a9ecd80-445c-11ea-868d-72d79cf4bf86.PNG)

Reply:
But I see your point. It might be better redoing it with winningCardToString()? selectCategoryToString()? cardToString()?

Reply:
No your way makes sense now. Leave it as it is!